content/_index.md

```diff
@@ -59,24 +58,20 @@ carousel:
 quotes:
```

Comment:
Did we plan on changing the logos on the homepage to link to the case studies when available?

Reply:
No reason not to! Will do.

Reply:
Thanks for changing this! My only suggestion now is to somehow visually indicate which one has a case study versus not. It very well may be that we want to move these to a completely different card structure -- with a teaser about what the case study concludes. Meantime, maybe just a badge of some sort?